core/sass/msg.scss

```diff
@@ -39,7 +39,7 @@
   -moz-border-radius:7px;
   -webkit-border-radius:7px;
   border-radius:7px;
-  font-size:1em;
+  font-size:10em;
```

Comment:
Not even a little bit. Lol. That was you and I testing to see if it worked.
1em is just lovely. Though 10 em would get us a rather entertaining
reaction from Trevor....


On Wed, Apr 15, 2015 at 9:58 PM, Ian Hunter <notifications@github.com>
wrote:

> In core/sass/msg.scss
> <https://github.com/BYU-ODH/hlrdesk/pull/245#discussion_r28482589>:
>
> > @@ -39,7 +39,7 @@
> >    -moz-border-radius:7px;
> >    -webkit-border-radius:7px;
> >    border-radius:7px;
> > -  font-size:1em;
> > +  font-size:10em;
>
> Is this still supposed to be 10em?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-ODH/hlrdesk/pull/245/files#r28482589>.
>